code/src/java/pcgen/io/PCGVer2Parser.java

```diff
@@ -6170,25 +6056,23 @@ private void resolveLanguages()
 					{
 						Collection<Language> selected =
 								(Collection<Language>) ss.getSet(thePC);
-						for (Language l : selected)
+						selected.stream().filter(cachedLanguages::contains).forEach(l ->
```

Comment:
I don't think we want full method behaviors inside parenthesis like this.  My 2c